src/setuptools_pyproject_migration/__init__.py

```diff
@@ -91,8 +91,50 @@ def _generate(self) -> Pyproject:
         if dependencies:
             pyproject["project"]["dependencies"] = dependencies
 
+        entrypoints = self._generate_entrypoints()
+
+        # GUI scripts and console scripts go separate in dedicated locations.
+        if "console_scripts" in entrypoints:
+            pyproject["project"]["scripts"] = entrypoints.pop("console_scripts")
+
+        if "gui_scripts" in entrypoints:
+            pyproject["project"]["gui-scripts"] = entrypoints.pop("gui_scripts")
+
+        # Anything left over gets put in entry-points
+        if entrypoints:
+            pyproject["project"]["entry-points"] = entrypoints
+
         return pyproject
 
+    def _parse_entrypoint(self, entrypoint: str) -> Dict[str, str]:
```

Comment:
```suggestion
    @staticmethod
    def _parse_entrypoint(entrypoint: str) -> Dict[str, str]:
```

Reply:
Yeah, good point… that'll actually make testing the method easier since I don't need to instantiate the class.